lib/locales/en.ts

```diff
@@ -534,6 +550,8 @@ export const locale: Locale = {
     ivan_merz: 'Blessed Ivan Merz',
     ivo_of_kermartin_priest: 'Saint Ivo, Priest',
     jacinta_marto_and_francisco_marto: 'Saints Jacinta and Francisco Marto',
+    jacques_francois_lefranc_martyr_and_martyrs_of_the_revolution:
+      'Blessed François Lefranc and the Martyrs of the Revolution',
```

Comment:
I think that in general, nobody knows what revolution is this. We should be more specific.

```suggestion
    jacques_francois_lefranc_martyr_and_martyrs_of_the_french_revolution:
      'Blessed François Lefranc and the Martyrs of the French Revolution',
```